### PR TITLE
Added datasource prometheus values for json data

### PR DESCRIPTION
**Pull request by kalinon**

### CODE REVIEW

File: datasource_test.go [hunk 1]
```diff
@@ -73,3 +73,34 @@ func TestNewDataSource(t *testing.T) {
 		t.Error("datasource creation response should return the created datasource ID")
 	}
 }
+
+func TestNewPrometheusDataSource(t *testing.T) {
+	server, client := gapiTestTools(200, createdDataSourceJSON)
+	defer server.Close()
+	//client_acc, _ := New("admin:admin", "http://localhost:3000")
```
Comment (mlclmj) [inline datasource_test.go]: Do the commented lines need to be in here?

File: datasource_test.go [hunk 2]
```diff
@@ -73,3 +73,34 @@ func TestNewDataSource(t *testing.T) {
...
+		IsDefault: true,
+		JSONData: JSONData{
+			HttpMethod:   "POST",
+			QueryTimeout: "60s",
+			TimeInterval: "1m",
+		},
+	}
+
+	created, err := client.NewDataSource(ds)
+	//created, err := client_acc.NewDataSource(ds)
```
Comment (mlclmj) [inline datasource_test.go]: Same as above

Comment (kalinon) [reply datasource_test.go]: Removed

File: datasource_test.go (hunk 1 above)
Comment (kalinon) [reply datasource_test.go]: Removed

**Pull request merged**